packages/perseus/src/components/image-loader.tsx

```diff
@@ -142,11 +142,18 @@ class ImageLoader extends React.Component<Props, State> {
         }
         const staticUrl = getDependencies().staticUrl;
 
+        // If the image is interactive, it should have a role of "button"
+        // to indicate that it is interactive.
+        const imageRole = imgProps.onClick !== null ? "button" : "img";
```

Comment:
I'm not positive that the onClick is ever null, but it seemed good to have this check in case there is a situation where the image ISN'T interactive. This matches the null check in the onClick logic further up in the file.